content/docs/user-guide/packaging/_index.md

```diff
@@ -14,5 +14,6 @@ The topics in this section provide information about how to prepare and package
 | [Creating a Project Game Release Layout for Windows](windows-release-builds) | Learn how to create an O3DE project game release layout for Windows. |
 | [Creating a Content Patch Update Package](content-patch-package) | Learn how to create updated asset bundles for content patches and release updates. |
 | [Project Export CLI Tool](project-export-cli) | Learn how to use the Project Export CLI to automate preparing your project for release. |
+| [Project Export for iOS](project-export-ios) | (Experimental) Learn how to use the Project Export CLI to create an Xcode project to build and deploy for iOS. |
```

Comment:
In general, why do we have a separate page here for iOS. The content should be placed in the same page as the project export cli (you can still mark the section experimental).

Reply:
Or at least make it a sub-page of `project-export-cli` instead of `packaging`

Reply:
I still think it's better as a separate page, since it'll contain only iOS specific content. So I'll change it to be a sub-page instead.